examples/plain-ruby/neural.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../helpers'
+
+# An executable specification that demonstrates how to use the Nylas Ruby SDK to interact with the API. It
+# follows the rough structure of the [Nylas API Reference](https://docs.nylas.com/reference).
+api = Nylas::API.new(app_id: ENV['NYLAS_APP_ID'], app_secret: ENV['NYLAS_APP_SECRET'],
+                     access_token: ENV['NYLAS_ACCESS_TOKEN'])
+
+message_id = api.messages.first.id
+
+# Perform sentiment analysis on a string
+sentiment = api.neural.sentiment_analysis_text("Hello world")
+demonstrate { sentiment.to_h }
+
+# Perform sentiment analysis on a message
+sentiments = api.neural.sentiment_analysis_message([message_id])
+demonstrate { sentiments[0].to_h }
+
+# Perform extracting a signature and parsing its contact information
+signatures = api.neural.extract_signature([message_id])
+demonstrate { signatures[0].to_h }
+demonstrate { signatures[0].to_h }
+# Convert the parsed contact to a Nylas contact object
+contact = signatures[0].contacts.to_contact_object
+demonstrate { contact.to_h }
+
+# Perform OCR request on a page (with a page range)
+file_id = api.files.first.id
+ocr = api.neural.ocr_request(file_id, [2,3])
+demonstrate { ocr.to_h }
```

Comment:
DIdn't work for me since I don't have any files. Maybe skip this section and print a warning if none exist? (could be a future TODO)

Reply:
Added a rescue block to prevent a fatal error in case of no file (or a non-document type of file was used) to prevent this from happening, thanks again!